account/urls.py

```diff
@@ -2,6 +2,6 @@
 from .views import SignUpView, SignInView
 
 urlpatterns = [
-    path('/sign-up', SignUpView.as_view()),
-    path('/sign-in', SignInView.as_view()),
+    path('sign-up', SignUpView.as_view()),
+    path('sign-in', SignInView.as_view()),
```

Comment:
```suggestion
    path('/sign-up', SignUpView.as_view()),
    path('/sign-in', SignInView.as_view()),
```

각 앱 url에는 앞에 슬래시를 붙이는 것이 wecode coding convention 입니다.